### PR TITLE
Update Stockquote Service and Working With Salesforce Client module.md files

### PR DESCRIPTION
**Pull request by Nipunich**
## Purpose
Update Stockquote Service and Working With Salesforce Client module.md files

### CODE REVIEW

File: docs/content/src/learn/tutorials/integration-patterns-and-soa/exposing-soap-service/src/stockquote_service/Module.md [hunk 1]
```diff
@@ -1 +1,44 @@
-# REST to SOAP
+Template for REST to SOAP Conversion
+
+# REST to SOAP 
+
+This is a template for the the tutorial described in [REST to SOAP](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/integration-patterns-and-soa/exposing-soap-service/1/). Please refer it for more details on what you are going to build here. This template provides a starting point for your scenario.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/integration-patterns-and-soa/exposing-soap-service/src/stockquote_service/Module.md]:
```suggestion
This is a template for the [REST to SOAP tutorial](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/integration-patterns-and-soa/exposing-soap-service/1/). Please refer to it for more details on what you are going to build here. This template provides a starting point for your scenario. 
```

File: docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md [hunk 2]
```diff
@@ -1 +1,47 @@
-# Working with Salesforce client
+Template for Working with Salesforce client
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md]:
```suggestion
Template for Working with Salesforce Client
```

File: docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md [hunk 3]
```diff
@@ -1 +1,47 @@
...
+
+# Working with Salesforce Client
+
+This is a template for the the tutorial described in [Working with Salesforce client](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/1/). Please refer it for more details on what you are going to build here. This template provides a starting point for your scenario.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md]:
```suggestion
This is a template for the [Working with Salesforce Client tutorial](https://ei.docs.wso2.com/en/7.0.0/ballerina-integrator/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/1/). Please refer it for more details on what you are going to build here. This template provides a starting point for your scenario.
```

File: docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md [hunk 4]
```diff
@@ -1 +1,47 @@
...
+$ ballerina new working-with-salesforce-client
+```
+
+Now navigate into the above module directory you created and run the following command to apply the predefined template you pulled earlier.
+
+```bash
+$ ballerina add -t wso2/working_with_salesforce_client working_with_salesforce_client
+```
+
+This automatically creates a working_with_salesforce_client for you inside the `src` directory of your project.
```
Comment (samgnaniah) [inline docs/content/src/learn/tutorials/saas-integrations/sfdc46/working-with-salesforce-client/src/working_with_salesforce_client/Module.md]:
```suggestion
This automatically creates a working_with_salesforce_client project for you inside the `src` directory of your project.  
```

**Comment by Nipunich** [comment]
This is modified in https://github.com/wso2/ballerina-integrator/pull/535

**Pull request closed without merging**